spec/interface.dd

```diff
@@ -4,8 +4,9 @@ $(SPEC_S Interfaces,
 
 $(HEADERNAV_TOC)
 
-    $(P An $(I Interface) describes a list of functions that a class which inherits
-    from the interface must implement.)
+    $(P An $(I Interface) abstracts from a common set of behaviour. It's an abstract class
+    which doesn't expose implementation details of methods and object state.
```

Comment:
This and the previous sentence should probably be joined together. The "it's" is a bit jarring.

Reply:
Maybe: "An Interface abstracts from behaviour in terms of an abstract class which ..."

I originally intended to use simple English but less text is surely reasonable as long as it's comprehensible.

If you have write access to this PR, you are welcome to do the change. I can do this only later on another device.

Reply:
@maxhaton I integrated your changes with my ones.